### PR TITLE
make event list more compact on mobile

### PR DESCRIPTION
**Pull request by marcelkalveram**
This is a relatively simple, initial approach to making the mobile site more compact.

I didn't let the image float next to the title because I think it gives the title too little space. Especially with longer titles, they'll wrap to the next line, making it look more awkward.

Happy to hear your suggestions to improve this further.

**Here are two screenshots for preview:**

iPhone X: https://www.dropbox.com/s/tub1pb849kkzz12/Screenshot%202019-10-08%2022.57.51.png?dl=0

Pixel XL: https://www.dropbox.com/s/9pdtgor68vlauyr/Screenshot%202019-10-08%2022.58.04.png?dl=0

### CODE REVIEW

File: data/assets/css/components/_event-summary.scss [hunk 1]
```diff
@@ -14,9 +14,12 @@
 .event-summary {
   position: relative;
   justify-self: left;
-  margin-bottom: $space-large;
+  width: 100%;
+  margin-bottom: $space-large/2;
```
Comment (hell03610) [inline data/assets/css/components/_event-summary.scss]: instead of dividing, try it with @space-medium - https://github.com/VLCTechHub/VLCTechHub-site/blob/a6e70150326410852befa87199816ca6dc8a8e7a/data/assets/css/abstract/_variables.scss#L12

File: data/assets/css/components/_event-summary.scss [hunk 2]
```diff
@@ -67,6 +71,9 @@
     height: 2px;
     color: black;
     background-color: black;
-    margin: $space-large auto $space-larger auto;
+    margin: $space-large/2 auto $space-larger/2 auto;
```
Comment (hell03610) [inline data/assets/css/components/_event-summary.scss]: same here, try with $space-medium in both or medium - large instead of the divisions, thx!

**Review by hell03610 — CHANGES_REQUESTED** [review]
I'm more inclined for a list for small screens with a design more like the one in the mobile app (maintaining the general styles of the web for links, separators, etc), but that will probably require also some changes in the html strucutre, so that the medium/large screens dont get affected.

But all in all, your proposal is a better design than the one we currently have, so if you want to go with it, go ahead after solving the code change suggested. Thanks a lot!!

**Comment by marcelkalveram** [comment]
Thanks for the feedback @hell03610 . I'll fix the two comments.

Agree that it would be good to be consistent, but playing a bit with the layout, I realised that I actually like the idea of having the avatar sitting on top of the title.

I'll give that another try and get back to you here.

**Comment by marcelkalveram** [comment]
Would you rather prefer something like this?

I personally don't dislike it, but I find it a tiny bit too squeezed, given the horizontal margin which doesn't exist in this way in the app.

Screenshot 2019-10-10 22 52 26

**Comment by hell03610** [comment]
I like it! I would try with smaller avatars and slightly less margin around the separator line, so there is more space for the title - but I like it!!

**Comment by marcelkalveram** [comment]
I've pulled in the latest changes and applied the changes (smaller avatar, less margin). I'm not sure I understood the idea about reducing the margin around the separator line. I guess you're referring to the `<hr>` and vertical margins around it?

What do we do with the new badge? (hoy, esta semana). I think it could go above the title, but as a smaller text, maybe?

Screen Shot 2019-10-15 at 14 15 00

**Comment by hell03610** [comment]
Thanks @marcelkalveram! Yes, the today/this week label was a pr this week. I would simply hide it in mobile for now and go ahead with this pull request. I'm really looking to merging it, the design looks very nice!

**Comment by marcelkalveram** [comment]
@hell03610 done!

File: data/assets/css/components/_event-summary.scss [hunk 3]
```diff
@@ -32,22 +37,36 @@
...
+          color: $brand-color;
+          background-color: $black;
+        }
       }
     }
 
     .reset-link {
       margin-bottom: $space-small;
       background-image: none;
+      float: left;
```
Comment (hell03610) [inline data/assets/css/components/_event-summary.scss]: could you use flex instead of float?

File: data/assets/css/components/_event-summary.scss [hunk 4]
```diff
@@ -77,13 +98,22 @@
     text-align: left;
     color: $text-color-secondary;
     font-size: 0.9rem;
+    margin-left: $thumbnailSizeMobile;
```
Comment (hell03610) [inline data/assets/css/components/_event-summary.scss]: I hope that with flex/grid instead of float this margin-left is not needed. But if it is truly needed, could you move it to https://github.com/VLCTechHub/VLCTechHub-site/blob/master/data/assets/css/abstract/_variables.scss in kebab-case?

**Review by hell03610 — CHANGES_REQUESTED** [review]
Interface looks very nice, but the current codebase does not use floats and I'd like to keep it that way, could you rephrase it to use flex or grid instead? thx!

File: data/assets/css/components/_event-summary.scss (hunk 3 above)
Comment (marcelkalveram) [reply data/assets/css/components/_event-summary.scss]: I'll work on that this week. I think because of the nesting, `float` was the easier solution, but I'll see if we can use flexbox instead.

File: data/assets/css/components/_event-summary.scss (hunk 4 above)
Comment (marcelkalveram) [reply data/assets/css/components/_event-summary.scss]: I'll take a look as soon as the float issue is resolved
